src/moonlink_backend/src/lib.rs

```diff
@@ -34,7 +38,9 @@ where
     // # Arguments
     //
     // * metadata_store_uris: connection strings for metadata storage database.
-    pub async fn new(base_path: String, metadata_store_uris: Vec<String>) -> Result<Self> {
+    //
+    // TODO(hjiang): [`_metadata_store_uris`] is not needed for moonlink self-managed database.
+    pub async fn new(base_path: String, _metadata_store_uris: Vec<String>) -> Result<Self> {
```

Comment:
I will take `Box<dyn BaseMetadataStore>` in the followup PR, which requires pg_mooncake side change.